Makefile

```diff
@@ -8,43 +8,24 @@ default: test-all
 test-all: test-library test-examples
 
 test-library-debug:
-	xcodebuild test \
-		-workspace ComposableArchitecture.xcworkspace \
-		-scheme ComposableArchitecture \
-		-destination platform="$(PLATFORM_IOS)"
-	xcodebuild test \
-		-workspace ComposableArchitecture.xcworkspace \
-		-scheme ComposableArchitecture \
-		-destination platform="$(PLATFORM_MACOS)"
-	xcodebuild test \
-		-workspace ComposableArchitecture.xcworkspace \
-		-scheme ComposableArchitecture \
-		-destination platform="$(PLATFORM_TVOS)"
-	xcodebuild \
-		-workspace ComposableArchitecture.xcworkspace \
-		-scheme ComposableArchitecture \
-		-destination generic/platform=watchOS
+	for scheme in ComposableArchitecture Dependencies; do \
+	  for platform in "$(PLATFORM_IOS)" "$(PLATFORM_MACOS)" "$(PLATFORM_TVOS)" "generic/platform=watchOS"; do \
+			xcodebuild test \
+				-workspace ComposableArchitecture.xcworkspace \
+				-scheme $$scheme \
+				-destination platform="$$platform"; \
+	  done; \
+	done
```

Comment:
I'm not very good with makefile but think this is easier to deal with than listing everything out manually.

Reply:
@mbrandonw Can -configuration be incorporated into it as well?

Reply:
Yeah it def could, but we actually decided to keep debug/release as separate steps in CI so that it's easier to see when a failure happened due to a release problem vs a common test failure.

Reply:
Good call :)